tests/e2e/specs/passcode/obscure-mode.test.ts

```diff
@@ -0,0 +1,82 @@
+import {expect} from '@wdio/globals';
+import {describe, it} from 'mocha';
+import {byText, byTextMatches, byResourceId} from '../../utils/selectors';
+import {output} from '../../utils/naming';
+
+describe('Passcode - Obscure Passcode Mode', () => {
+  it('should show a blank Observations screen after entering obscure passcode', async () => {
+    const obsListTab = await $('~Go to ObservationsList');
+    await obsListTab.click();
+    await expect($(byTextMatches('Lake'))).toBeDisplayed();
+    await driver.terminateApp('com.comapeo.rc');
+    await driver.activateApp('com.comapeo.rc');
```

Comment:
I don't know if we can change this, but this means these test will only work on RC builds, so we wont be able to run these tests on dev builds like andrew mentioned. I think it is fine for now, since we don't know if we can run e2e on dev build but just noting so we know @achou11